### PR TITLE
[14.0] [MIG] account_menu_invoice_refund

### PR DESCRIPTION
**Pull request by khushbu-3**

### CODE REVIEW

File: account_menu_invoice_refund/views/account_invoice_view.xml [hunk 1]
```diff
@@ -7,8 +7,8 @@
         <field name="res_model">account.move</field>
         <field name="view_mode">tree,form</field>
```
Comment (RodrigoBM) [inline account_menu_invoice_refund/views/account_invoice_view.xml:8]: add kanban mode

File: account_menu_invoice_refund/views/account_invoice_view.xml [hunk 2]
```diff
@@ -7,8 +7,8 @@
...
         <field eval="False" name="view_id" />
-        <field name="domain">[('type', 'in', ('out_invoice', 'out_refund'))]</field>
-        <field name="context">{'type':'out_invoice', 'journal_type': 'sale'}</field>
+        <field name="domain">[('move_type', 'in', ('out_invoice', 'out_refund'))]</field>
+        <field name="context">{'move_type':'out_invoice', 'journal_type': 'sale'}</field>
```
Comment (RodrigoBM) [inline account_menu_invoice_refund/views/account_invoice_view.xml:11]: change move_type to default_move_type. i think that journal_type is not required

File: account_menu_invoice_refund/views/account_invoice_view.xml [hunk 3]
```diff
@@ -43,8 +43,8 @@
         <field name="res_model">account.move</field>
         <field name="view_mode">tree,form</field>
```
Comment (RodrigoBM) [inline account_menu_invoice_refund/views/account_invoice_view.xml:44]: add kanban mode

File: account_menu_invoice_refund/views/account_invoice_view.xml [hunk 4]
```diff
@@ -43,8 +43,8 @@
...
         <field eval="False" name="view_id" />
-        <field name="domain">[('type', 'in', ('in_invoice', 'in_refund'))]</field>
-        <field name="context">{'type':'in_invoice', 'journal_type': 'purchase'}</field>
+        <field name="domain">[('move_type', 'in', ('in_invoice', 'in_refund'))]</field>
+        <field name="context">{'move_type':'in_invoice', 'journal_type': 'purchase'}</field>
```
Comment (RodrigoBM) [inline account_menu_invoice_refund/views/account_invoice_view.xml:47]: change move_type to default_move_type. i think that journal_type is not required

**Comment by AaronHForgeFlow** [comment]
Superseded here: https://github.com/OCA/account-invoicing/pull/1347

**Pull request closed without merging**